payload_bay_micro.ino

```diff
@@ -12,6 +12,10 @@ const unsigned long RADIO_FREQ = 20;
 // The last time the altitude was transmitted, in us
 unsigned long lastRadioTime = 0;
 
+bool turnedOff = true;
+
+unsigned long time;
```

Comment:
These could be declared as `static` variables inside `loop()`, like `mode` is, but ultimately it doesn't matter.

To be clear, the only difference between a `static` variable inside a function and a regular global variable is that the global variable can be accessed inside other functions.